code/modules/surgery/surgery_step.dm

```diff
@@ -93,7 +93,10 @@
 	var/was_sleeping = (target.stat != DEAD && target.IsSleeping())
 
 	// Skyrat Edit Addition - reward for doing surgery on calm patients, and for using surgery rooms(ie. surgerying alone)
-	if(was_sleeping || HAS_TRAIT(target, TRAIT_NUMBED) || target.stat == DEAD)
+	if(!(was_sleeping || HAS_TRAIT(target, TRAIT_NUMBED) || target.stat >= UNCONSCIOUS))
```

Comment:
I think this is what you meant to do, otherwise it would be really cursed.
Also in general I feel like you should do one check for the was_sleeping or if they're at least unconscious, I don't know, this feels like a lot of code repetition.
```suggestion
	if(!(was_sleeping || HAS_TRAIT(target, TRAIT_NUMBED) || target.stat < UNCONSCIOUS))
```